src/main/kotlin/com/zim/pokemon_api/model/Pokemon.kt

```diff
@@ -0,0 +1,25 @@
+package com.zim.pokemon_api.model
+
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "pokemons")
+data class Pokemon(
+    @Id
+    @Column(name = "id")
+    val id: Int,
```

Comment:
Btw, why not use the pokedex number as id? 
Or somehow marry those two together, they seem to be equal.